03.asynchronous/callback/callback.js

```diff
@@ -0,0 +1,18 @@
+import sqlite3 from "sqlite3";
+
+const db = new sqlite3.Database(":memory:");
+
+const CREATE_TABLE =
+  "CREATE TABLE books (id integer primary key autoincrement, title text not null unique)";
+const INSERT_RECORD = "INSERT INTO books (title) VALUES(?)";
```

Comment:
他のスタイルを揃えるなら以下のようにしたほうがよいと思います。

```suggestion
const INSERT_RECORD = "INSERT INTO books (title) VALUES (?)";
```

Reply:
こちら修正しました!!
72f6a6c525d266ee4227db954b430a25523d34b0
定数化したクエリも別ファイルからインポートするようにしたので、定義場所が移動しました。